streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -1447,6 +1450,11 @@ private int updateMinSupportedVersion(final int supportedVersion, final int minS
         }
     }
 
+    private String formatInstantTime(final long instant) {
```

Comment:
As you suggested, this function would be a better fit for a Util class because it's being used twice across two different classes.

Perhaps rename it to `ToLogDateTimeFormat()` and add it to a new class `commons.utils.LoggingUtils`